tests/python_tests/test_tokenizer.py

```diff
@@ -220,22 +236,27 @@ def test_set_chat_template():
     'Why is the Sun yellow?',
     'What was my first question?',
     ['Why is the Sun yellow?'],
-    "若我有一亿美元，在人工智能盛行的今天，我怎样投资才能收益最大化？",
```

Comment:
ov_genai.Tokenizer was truncating to 77 tokens length while hf did not that by default, therefore shortened prompt a little bit.

Padding issues are under work in a separate PR.